docs/learn/software_overview.md

```diff
@@ -132,13 +135,13 @@ card. You can put the file on the SD card, and set the whole thing up on the LCD
 without bringing a computer into the shop.
 
 There are many other advanced ways to send gcode. You can hook up a raspberry pi, and use the [v1pi
-image](../electronics/v1pi.md). You can use CNC.js standalone on your computer, or use pronterface.
+image](../electronics/v1pi.md). You can use [CNC.js](https://github.com/cncjs/cncjs/wiki/Electron-App) standalone on your computer, or use pronterface.
 
-## Controller
+## Milling-Controller-Firmware
```

Comment:
I am ok with adding firmware, but I would scratch milling. Also, we don't need the '-' marks. I can fix this, and the links, if you like.

These titles end up in the table of contents, which doesn't show up as well in github. But they need to be skimmable and easily readable.

Reply:
Agreed.  What I tried to do is basically name each section title (perhaps unsuccessfully) as such:

Workflow Step/Software Used (e.g. Design/CAD).

However I could not get the slashes to work with links in the markup.  I changed them to dashes knowing you guy would say "dashes are ugly, here is what we will change them to".

Reply:
```suggestion
## Controller
```